XEVIOUS/SceneManager.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Scene.h"
+#include "enum_Scene.h"
+
+class Scene;
+
+class SceneManager
+{
+public:
+	SceneManager();
+	~SceneManager();
+	void Update();
+	void Render();
+
+private:
+	static Scene*	m_pScene;
```

Comment:
```suggestion
	static Scene* m_pScene;
```
スペースじゃなくてタブ文字が入ってませんか？

Reply:
なぜ`static`？

Reply:
staticで定義したらその変数の中身がどのインスタンスでも同じと聞いたのでstatic付けたのですが、シーンが切り替わる時にm_pSceneをdeleteした直後にすぐ次のシーンをnewするので中身が同じとか意味がなかったです
staticを外しても処理が同じだったので外しときます